backitup.sh

```diff
@@ -168,14 +168,9 @@ if [ $BACKUP_TYPE == "minimal" ]; then
 
 #    Backup ausfuehren
     echo "--- Minimal Backup started ---"
-    iobroker backup && echo success "--- Minimal Backup created ---" || echo error "--- Cannot create minimal backup ---"
+    iobroker backup minimal$NAME_SUFFIX-$date-$time$FILENAME_ADDITION.tar.gz && echo success "--- Minimal Backup created ---" || echo error "--- Cannot create minimal backup ---"
```

Comment:
You can provide the name directly to command.
If not, you can get the problem if the backup is longer than one minute and you start to do it in the last minute of the hour.